src/roletype.ts

```diff
@@ -0,0 +1,5 @@
+export enum roletype {
+    SADMIN = 'SuperAdmin',
```

Comment:
When using enum, the name should be complete. In order to understand the purpose. 
SUPER_ADMIN